pkg/platform/kube/platform.go

```diff
@@ -261,13 +261,6 @@ func (p *Platform) CreateFunction(ctx context.Context, createFunctionOptions *pl
 			briefErrorsMessage = p.clearCallStack(briefErrorsMessage)
 		}
 
-		// low severity to not over log in the warning
-		createFunctionOptions.Logger.DebugWithCtx(ctx, "Function creation failed, brief error message extracted",
-			"briefErrorsMessage", briefErrorsMessage)
```

Comment:
leave this log, with the brief error message (will only be visible in the pod logs)